pkg/controller/operators/operatorcondition_controller_test.go

```diff
@@ -313,6 +313,44 @@ var _ = Describe("OperatorCondition", func() {
 							Value: operatorCondition.GetName(),
 						}))
 					}
+
+					Context("if the OPERATOR_CONDITION_NAME environment variable is removed", func() {
```

Comment:
I don't think `Context` is typically found inside `It` blocks. Is is possible to refactor this to be more [idiomatic](https://onsi.github.io/ginkgo/#organizing-specs-with-containers-describe-and-context)?

Reply:
Can do!